src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,50 +1,95 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (movie:Movie) => void;
+};
+
+export const NewMovie:React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+
+  const checking = title && imgUrl && imdbUrl && imdbId;
```

Comment:
```suggestion
  const isSubmitEnabled = title && imgUrl && imdbUrl && imdbId;
```
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/96409563/d404d860-9b10-47b8-8a81-9fad09752475)